packages/concerto-core/lib/decoratormanager.js

```diff
@@ -813,3 +813,4 @@ class DecoratorManager {
 }
 
 module.exports = DecoratorManager;
+module.exports.DCS_MODEL = DCS_MODEL;
```

Comment:
```suggestion
```

Reply:
if we export this like this
`module.exports = { DCS_MODEL, DecoratorManager };`
then the DecoratorManager isn't exported as default which is expected by tests so tests will fail .